Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -1390,6 +1433,51 @@ class CloneSerializer : CloneBase {
                 write(String::fromLatin1(JSC::Yarr::flagsString(regExp->regExp()->flags()).data()));
                 return true;
             }
+            if (auto* errorInstance = jsDynamicCast<ErrorInstance*>(obj)) {
+                auto& vm = m_lexicalGlobalObject->vm();
+                auto scope = DECLARE_THROW_SCOPE(vm);
+                auto errorTypeString = errorInstance->get(m_lexicalGlobalObject, vm.propertyNames->name).toWTFString(m_lexicalGlobalObject);
+                RETURN_IF_EXCEPTION(scope, false);
```

Comment:
Let's error check twice. `get` can throw error, and can return JSEmpty. And `toWTFString` onto JSEmpty will crash. So,

```
auto errorTypeValue = errorInstance->get(m_lexicalGlobalObject, vm.propertyNames->name);
RETURN_IF_EXCEPTION(scope, false);
auto errorTypeString = errorTypeValue.toWTFString(m_lexicalGlobalObject);
RETURN_IF_EXCEPTION(scope, false);
```

Reply:
Ok.